port/unix/omrsysinfo.c

```diff
@@ -3706,13 +3817,14 @@ int32_t
 omrsysinfo_cgroup_get_memlimit(struct OMRPortLibrary *portLibrary, uint64_t *limit)
 {
 	int32_t rc = OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM;
+
+	Assert_PRT_true(NULL != limit);
```

Comment:
Please update the function prototype to indicate `limit` cannot be null.

Reply:
Done